metrics/ci_integrations/test/source/jenkins/adapter/jenkins_source_client_adapter_test.dart

```diff
@@ -134,7 +139,7 @@ void main() {
           fetchLimit,
         );
 
-        expect(result, completion(equals(expected)));
+        expect(result, completion(hasLength(jenkinsBuilds.length)));
```

Comment:
Make `async` and `await` the result. Then:
```suggestion
        expect(result, hasLength(jenkinsBuilds.length));
        expect(result, contains(isNot(isInProgressBuild)));
        expect(result, contains(isInProgressBuild));
```
Or create the proper expected list and:
```suggestion
        expect(result, equals(expected));
```